scipy/stats/boost/include/code_gen.py

```diff
@@ -122,7 +121,7 @@ def ufunc_gen(wrapper_prefix: str, types: list, num_ctor_args: int, filename: st
             for jj, t in enumerate(types):
                 ctype = {
                     'NPY_LONGDOUBLE': 'longdouble',
-                    'NPY_DOUBLE': 'double',
+                    'NPY_DOUBLE': 'double ',
```

Comment:
I'm curious - is this intentional?

Reply:
Nope, I was testing to see if changing would force a rebuild and forgot to change back.